test/e2e/workspacetemplate_test.go

```diff
@@ -248,52 +250,39 @@ spec:
 
 	Context("Template Validation", func() {
 		It("should reject workspace with image not in allowlist", func() {
-			By("verifying test-rejected-workspace was created from test-template-rejection.yaml")
-			verifyWorkspaceExists := func(g Gomega) {
-				cmd := exec.Command("kubectl", "get", "workspace", "test-rejected-workspace",
-					"-o", "jsonpath={.metadata.name}")
-				output, err := utils.Run(cmd)
-				g.Expect(err).NotTo(HaveOccurred())
-				g.Expect(output).To(Equal("test-rejected-workspace"))
-			}
-			Eventually(verifyWorkspaceExists).
-				WithPolling(1 * time.Second).
-				WithTimeout(10 * time.Second).
-				Should(Succeed())
-
-			By("waiting for Valid condition to be False")
-			verifyTemplateRejected := func(g Gomega) {
-				cmd := exec.Command("kubectl", "get", "workspace", "test-rejected-workspace",
-					"-o", "jsonpath={.status.conditions[?(@.type==\"Valid\")].status}")
-				output, err := utils.Run(cmd)
-				g.Expect(err).NotTo(HaveOccurred())
-				g.Expect(output).To(Equal("False"))
-			}
-			Eventually(verifyTemplateRejected).
-				WithPolling(1 * time.Second).
-				WithTimeout(10 * time.Second).
-				Should(Succeed())
-
-			By("verifying Degraded condition is False (policy violation is not system degradation)")
-			cmd := exec.Command("kubectl", "get", "workspace", "test-rejected-workspace",
-				"-o", "jsonpath={.status.conditions[?(@.type==\"Degraded\")].status}")
+			By("attempting to create workspace with invalid image")
+			cmd := exec.Command("sh", "-c", `echo 'apiVersion: workspace.jupyter.org/v1alpha1
```

Comment:
Could we please define this as a separate file vs inline? When #67 will be merged, let's reuse test folder and files as much as possible